src/components/TheFondue.vue

```diff
@@ -37,11 +46,21 @@ export default {
 <style scoped>
 .fondue {
 	position: relative;
-	background: url("@/assets/fondue.jpg") center;
-	background-size: cover;
 	height: 100vh;
 }
 
+.fondue picture {
+	position: absolute;
+	inset: 0;
+	overflow: hidden;
+}
+
+.fondue picture img {
+	width: 100%;
+	height: 100%;
+	object-fit: cover;
+}
```

Comment:
For comparison, using [CSS Nesting](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_nesting/Using_CSS_nesting):

```patch
- .fondue {
- 	position: relative;
- 	height: 100vh;
- }
- .fondue picture {
- 	position: absolute;
- 	inset: 0;
- 	overflow: hidden;
- }
- .fondue picture img {
- 	width: 100%;
- 	height: 100%;
- 	object-fit: cover;
- }

+ .fondue {
+ 	position: relative;
+ 	height: 100vh;
+ 
+ 	& picture {
+ 		position: absolute;
+ 		inset: 0;
+ 		overflow: hidden;
+ 
+ 		& img {
+ 			width: 100%;
+ 			height: 100%;
+ 			object-fit: cover;
+ 		}
+ 	}
+ }
```